### DISCUSSION TITLE
追加曲～と既存曲へのループタグの埋め込み

### DISCUSSION THREAD

**Discussion by ponyexs** [beta1139/ff7fc-develop#83] [General]
祝！ディスカッション初の新規作成w
<br>

必要か不要か邪魔かは考えず
とりあえず素材の一つとして作れるだけ作っていくことにしました
<br>
[2oggs.zip](https://github.com/beta1139/ff7fc-develop/files/5966608/2oggs.zip)
　※OGGファイルが直接ここに置けないようなのでZIPファイルにまとめています。
<br>
●1-08-Lurking_in_the_Darkness.ogg<br>
元のOGGにループタグを入れました。
<br>
<br>
●ルーファウス歓迎式典.ogg<br>
BGMフォルダ内に見当たらなかったのでとりあえず作成しました。
きちんとした曲が登場するまでの繋ぎになれば幸いです。

とりあえずサントラの名前をそのまま付けただけなので
ご利用いただける場合はすみませんが他のファイル命名方式（？）に合うようにファイル名を変更してください。
<br>

**Comment by ponyexs** [comment]
[エレキ・デ・チョコボ_f1.zip](https://github.com/beta1139/ff7fc-develop/files/5971340/_f1.zip)
(fix1: ループ位置がずれていたので修正)

もう1曲追加します
クラウド回想シーンの次はいよいよチョコボの出番ですね🐓
間に合わせですがチョコボ捕獲戦の曲です

きちんとした同曲が登場するまでの繋ぎのつもりなので
とりあえず曲の後半はカットして前半1分程度の部分を繰り返す形にしています😷

**Reply by i7-FF7FC** [reply]
> ルーファウス歓迎式典

うぉぉーすごく有り難い:heart_eyes:
繋ぎじゃなくても良い気がしますよw

このイベント作れるんだろか…？
私が鬼門だと思っている内の1つです。

**Reply by i7-FF7FC** [reply]
> 1-08-Lurking_in_the_Darkness.ogg

あれループタグ入れてなかったか。
ありがとうございます！差し替えておきますね。


> エレキ・デ・チョコボ_f1.zip

こちらも感謝です！
レッド加入曲の時もそうでしたが、
たぶん足りない曲あるよなぁー(どうやって誤魔化そう)と
何気に気になってたのでwメチャクチャ頼もしい！

ところで、このチョコボ捕獲イベどうしましょうね。
兼ねてよりチョコボ関連は全て未定で後回しにしてたんですが、
今やってる回想イベが終わったら、次はいよいよコレなんですよね。
いやーほんとどうすんだろ。。回想後半があるので、まだちょい先にはなりそうですが。

単純に原作と同じ、戦闘中に捕まえるシステムにするのか、
若しくはファミコンFFと同じ、チョコボの森で捕まえる仕様にするのか・・・:thinking:
いずれにせよ、チョコボに乗れるシステムをbetaさんに実装していただく必要が。

**Reply by ponyexs** [reply]
> 1-08-Lurking_in_the_Darkness.ogg

うーん、もしかしたらループタグは入っていたのかもしれませんが…
ゲーム内ではフェードアウトした後にまた頭から再生されていましたので
フェード箇所より前でループするようにした形です。
<br>

> 歓迎式典

クラウドの過去のピアノが実現させるほどですから
式典のボタンイベントなんて朝飯前でしょうっ（無責任🤩
<br>

> チョコボ🐓

確かに戦闘で勝った時に捕まえる、という仕組みを作るのはプログラムが大変そう…
そもそもチョコボよせマテリアを装備しているときだけチョコボが出る、
という仕組みを組むこと自体が大ごとな気もします…
（プログラムがわからないのでその大変さは理解できていませんが）

**Comment by ponyexs** [comment]
セフィロスの心情・行動を現すシーンで流れる2曲。

　🎵EX-12-Pre_Those_Chosen
　🎵3-05-Those_Chosen_by_the_Planet

原作では同じ曲のイントロと本編ですが
現在のBGM素材は音色が大きく変化するのが気になったので
同じ音色タイプとなるものを（需要の有無は関係なく勝手に）作りました。

　[2oggs_sephiroth_Plan-B.zip](https://github.com/beta1139/ff7fc-develop/files/5977906/2oggs_sephiroth.zip)

これが素材の選択肢の幅が広がる一助となれば幸いです。🙇

🤔
そういえばFF5の終盤にこれとよく似た曲があったなぁ・・・

**Reply by ponyexs** [reply]
すみません、さっそく修正です。


●エレキ・デ・チョコボ_f2
　fix2:
　イントロのチョコボの声を原作に近づけました
　ファミコン3和音のうち2和音分しか使っていなかった箇所に音を入れました


●3-05-Those_Chosen_by_the_Planet_plan-b_f1
　fix:1
　奏法を一部変えてたくさん音が鳴っているように見せかけました

●EX-12-Pre_Those_Chosen
　こちらはそのままです

[3oggs_sephiroth_Plan-B_fix.zip](https://github.com/beta1139/ff7fc-develop/files/5981029/3oggs_sephiroth_fix.zip)

**Reply by i7-FF7FC** [reply]
お疲れ様です！

> EX-12-Pre_Those_Chosen

ありゃ。。不自然でしたか:sweat_drops:
前半部分が無かったので、何となく合うようにはやってたつもりでしたが、
実際にイベにはめたことはなかったので、もっぺんトライしてみるかなー。
bpmも原作とほぼ同じ76ぐらいに合わせてたんですが、
3-05-Those_Chosen～と同じにした方が良いんだろか。

やはりファミコンベースなので、既にnsfバージョンがある場合は
第一希望としては、出来るだけそれを生かしたいというのがありまして、
なのでまた上手くいかなかったら、有難くお借り致しします:bow:

ところで、この前奏部分、
最後バスドラだけになる2小節ありませんでしたっけ？

**Reply by ponyexs** [reply]
すみません、比べて聞いてみると
FF7FC版の本編のほうは鐘の音がカットされているようですので
違うように感じるのかもしれません。

nsfで作られた曲があるということは知らなかったので、
全ての曲と音がPCのMIDIなどでファミコンに似せて作ってあるものなのかと思っておりました🙏
既に実際のファミコンとして使えるものがあるならそれに越したことはないかと思います。

> 最後バスドラだけになる2小節ありませんでしたっけ？

うーん、私の場合はPS版ではなくPC版のインターナショナルを遊んでいますが、
こちらでは鐘の音が混じる小節しかなさそうです。
オリジナルのPS版とはもしかしたら仕様が違うのかもしれませんが…

**Reply by i7-FF7FC** [reply]
> 鐘の音が混じる小節しかなさそうです。

いや私もインタ版ですが、例えば適当にWalkthroughを見ても、
こんな感じでバスドラ「ドドン」が計8回鳴っています。
(4回目からはバスドラのみになる)←訂正5回目
https://www.youtube.com/watch?v=TQdK4oGBNH0&list=PL85002125B482FF1B&index=25

**Reply by i7-FF7FC** [reply]
> nsfで作られた曲があるということは知らなかったので

既に元データがあった素材置き場が消滅してしまっているため
(完全に不覚でバックアップ取っておくべきでした:sob:)、
この「3-05-ThoseThose~」に関しては、正確にはそうであったかは定かではありませんが、
たぶん音聞く限りそうじゃないかなぁと。
誤解を恐れずに言うと、nsfの音って良い意味での独特の細さというかシャープさがあるので。
(分かりませんけど！)

ちなみに、フィールド曲とか、Anxious_Heartとかは元nsfですね。
数は数えていませんがそれなりにあると思います.
クレイジーモーターサイクルも実データあるので。

**Reply by ponyexs** [reply]
私のところはこれです。心音4回で鐘の音が1回です。

https://youtu.be/Z8rXs7Y-Xzc

スクエニのサイトでダウンロード販売されているFF7インター（PC版）です。
インター版でもPC版とPS版では異なるのかも？あるいは日本版と海外版？？🤔


MIDIからお手軽にnsfが変換できれば楽なんでしょうけれどね😷
それか楽譜を作る要領でNSFで曲が作れる作曲ソフトとかがあれば…！

**Reply by i7-FF7FC** [reply]
あれーほんとだー:confused:
個人的にこのバスドラだけになる部分が心音みたいで緊迫感があって好きだったんですが、
うーん・・・どうしよう・・・個人の好きに！:triumph:


> nsfが変換できれば

私も当時死ぬほど探しました。
そして無理だと分かりました！
なのでnsfサイトで勉強しました。
そして無理だと分かりました！

**Reply by ponyexs** [reply]
PC版はPS版からかなり時間が経過して発売されていますから
細かい変更点がいろいろあるのかもしれませんね。

普及率…といいますかFF7を遊んだ人の機種はPS版が圧倒的でしょうから
個人的には**違いがある場合はPS版を優先**した方がいいかなぁかとは思います

私はPC版のほうの知識で話をしたりモノを作ったりしているので
すり合わせは必要になるかもしれませんね

<br>

nsfの作成はファミトラッカーというソフトがいいと
以前教えてもらったことがありましたが…
まったく身に付きませんでした！😷

**Reply by ponyexs** [reply]
[EX-12-Pre_Those_Chosen_plan-b_beat8.zip](https://github.com/beta1139/ff7fc-develop/files/5988302/EX-12-Pre_Those_Chosen_plan-b_beat8.zip)
というわけでして、不要かもしれませんが一応
昨日教えていただいた動画に合わせて心音8回鐘1回のものも置いておきます。

…どうもゲームボーイやPCエンジンのように音が太くなるため
ファミコンのような繊細な感じにはならないですね…

↑書いていてちょっと腑に落ちたのですが、
私のはファミコン風というよりゲームボーイ風なのですね。

というわけで、今後ゲームボーイ風FF7の制作に取り掛かる際はぜひひとつこちらをw

**Reply by i7-FF7FC** [reply]
> 違いがある場合はPS版を優先

ではそうしますかー。
しかし面白いですねこういうのは。
これを作ってると、まず通常プレイでも気付かないようなことを定期的に発見できます。

この前のエレベーターもそうですし、
スラム教会で目覚める際に、暗転中ボタン連打しないと
「あ　うごいた」の台詞が表示されないとか:open_mouth:

**Reply by i7-FF7FC** [reply]
> EX-12-Pre_Those_Chosen_plan-b_beat8.zip

あざます！！:bow:
失敗しても保険があるので気軽に挑戦できますw

なるほどゲームボーイ風か。
自分はあまりそっちは明るくないのですが、
音の厚み・迫力は確かにponyexsさんの方があるんですよね。
一般的な音楽制作だと、大体太い方が正義になりがちなので、
この辺もファミコンワールドは独特だなと思います。

**Reply by ponyexs** [reply]
> 「あ　うごいた」

これボタン連打で変わる仕組みだったんですね
表示されるときとされないときがあったので単にランダムなのかと思ってました。
芸が細かいなーFF7🤩
<br>

> 一般的な音楽制作だと、大体太い方が正義になりがち

ああ、なるほど
言われてみれば、スクエニから発売されている
FF7のチップチューンCD（FINAL FANTASY VII CHIPS）は
音色こそファミコンに近いものがいくつかありましたが
音自体に厚みがあって和音もかなり多めでした

**Reply by i7-FF7FC** [reply]
「Those_Chosen_by_the_Planet」前奏込みで作り直しました:notes:
音量とか細かいところは変わるかも知れませんが。
https://ux.getuploader.com/FF7FC_up/download/56

今回サンプリング使っているので、繋がりに関しては違和感ないはず...
あと元のオリジナルが最後の白玉で伸ばしてるとこで、
一瞬プッていう録音ミスみたいなのが入ってたので修正しました。

**Reply by i7-FF7FC** [reply]
追記：
すみません、さっきの削除して、
鐘の音だけ3パターン用意してみました。
https://ux.getuploader.com/FF7FC_up/download/57

01:サンプリング
02:Famisynth
03:8bit-Plug

Bassは全て元音源サンプリングで、Kickは8bit-Plugです。

**Comment by ponyexs** [comment]
おおー
やっぱりイントロと本編とで差異を感じた正体は鐘の音の有無でした
01,02,03どれも良いですね！

…ありゃ選べない💦
よし
場面ごとに3タイプを使い分けるということで！ ﾄﾞｰﾝ😤

強いてあげるなら
音が強くなる本編に入ってもしっかり鐘が目立つ02famisynthが私の中では一番の候補かも

**Reply by i7-FF7FC** [reply]
ですねぇー。
音の素材もそうですが、フレーズも同じものが鳴っているっていうのが重要でしたね。
実際やってみて、ponyexsさんのご指摘は実にその通りだと思いました:sweat_smile:ありがとうございます。

そして奇遇ですねー。
私も02>01>03で、02が一番鐘っぽくて好きですw

ただ元のthose～との混ざり具合は、01が良い感じもするので、
その辺もうちょっと試して、最終的にイベに実装する際、
BGM2パターン用意してみる・・・かも？知れません。
その際はお手数ですが、またよろしくお願いします！:grinning:

**Reply by ponyexs** [reply]
承知しましたー
楽しみにしております🤩
